sdcm/cluster.py

```diff
@@ -2448,8 +2448,8 @@ def prepare_files_for_archive(self, fileslist):
                 self.remoter.run('cp -r %s %s' % (f, new_full_path), ignore_status=True)
         return root_dir
 
-    def generate_coredump_file(self, restart_scylla=True):
-        self.log.info('Generate scylla core')
+    def generate_coredump_file(self, restart_scylla=True, node: Any = None):
```

Comment:
node isn't Any, it should be BaseNode, otherwise there's no meaning for adding type annotations here

Reply:
perhaps it better be either a BaseNode or an STR?

Reply:
fixed